data/src/config/buffer.rs

```diff
@@ -52,15 +44,29 @@ impl ServerMessages {
     }
 }
 
-#[derive(Debug, Copy, Clone, Default, Deserialize)]
+#[derive(Debug, Copy, Clone, Deserialize)]
+#[serde(rename_all = "kebab-case")]
 pub struct ServerMessage {
+    #[serde(default = "default_bool_true")]
+    pub enabled: bool,
     #[serde(default)]
-    pub exclude: Exclude,
+    pub smart: Option<i64>,
     #[serde(default)]
     pub username_format: UsernameFormat,
```

Comment:
`username-format` should be `username_format` since we're going default rust field / snake_case?